app/containers/ITunesDeatails/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ * ITunesDeatails Container
+ *
+ */
+
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import T from '@components/T';
+import TrackDetails from '@app/components/TrackDetails/index';
+import { selectResults, selectError } from './selectors';
+import { selectTracks, selectTrackById } from '../ITunesProvider/selectors';
```

Comment:
Please use absolute imports when importing anything outside the container/component folder.